src/components/InfoPanel.component.js

```diff
@@ -27,19 +27,21 @@ class InfoPanel extends Component {
       activePOIs,
       isStorySelected,
       selectedPOIIndex,
+      previewedPOIIndex,
```

Comment:
and that would basically changed all instances of "previewed" to "previewing" (except for the action type/action creator)